demo/spacy_annotator_demo.ipynb

```diff
@@ -264,7 +264,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "df_labels"
+    "df_labels\n",
+    "annotator.to_spacy(df_labels, \"spacy_labels.spacy\")"
```

Comment:
I  would split line 267 and 268 in two separate cell blocks.
Also, I would change line 268 to:
```python
spacy_annotations = annotator.to_spacy(df_labels, "spacy_labels.spacy")
```